nilearn/_utils/fixes/sklearn_validation.py

```diff
@@ -399,3 +399,27 @@ def check_is_fitted(estimator, attributes, msg=None, all_or_any=all):
 
     if not all_or_any([hasattr(estimator, attr) for attr in attributes]):
         raise NotFittedError(msg % {'name': type(estimator).__name__})
+
+
+def indexable(*iterables):
```

Comment:
Sorry, please discard this comment. It was a mistake that I made with the diff tool.